experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/proxies.spec.ts

```diff
@@ -69,10 +69,11 @@ describe("SharedTreeObject", () => {
 	const parentSchema = sb.object("parent", {
```

Comment:
It would probably be worth adding a FluidHandle to this, since we added support in this layer for them. Should be fine to just us a `MockHandle`.

Reply:
Good idea, I added a read/write test for handles.